packages/terra-status-view/src/StatusView.jsx

```diff
@@ -53,7 +53,7 @@ const propTypes = {
    * `no-matching-results`, `not-authorized`,
    * `error`, or `custom`.
    */
-  variant: PropTypes.string,
+  variant: PropTypes.oneOf(['no-data', 'no-matching-results', 'not-authorized', 'error', 'custom']),
```

Comment:
Instead of hardcoding these values, could we do Object.values(StatusViewVariants)?

Reply:
Makes sense, I will check it out.

Reply:
Changed: [846faa8](https://github.com/cerner/terra-core/pull/2264/commits/846faa8916a54c4b847be71184220fb9a6fc14b3)